File: Python/IBM/Connection.py
import ibm_db


def conectar():
    conn = ibm_db.connect(
        "DRIVER={IBM DB2 ODBC Driver};"
        "HOSTNAME={SEU HOST};"
        "PORT={SUA PORTA};"
        "DATABASE={SEU DATABASE};"
        "UID={SEU USERNAME};"
        "PWD={SEU PASSWORD};"
    )

    return conn


def desconectar(conn):
    if conn:
        conn.close()

File: Julia/Mongo/Connection.jl
using Mongo


function conectar()
    conn = Mongo.connect("SEU HOST", 27017)

    return conn
end


function get_collection(conn)
    collection = conn |> Mongo.get_collection("SUA COLEÇÃO")
    
    return collection
end


function desconectar(conn)
    conn |> Mongo.close()
end